vcert/connection_cloud.py

```diff
@@ -625,8 +625,8 @@ def set_policy(self, zone, policy_spec):
         for_update, owners_list = self.resolve_owners(policy_spec.users, user_details)
         if app_details:
             # Application exists. Update with cit and owners
-            if for_update and len(owners_list) > 0:
-                log.debug("No owners provided in the policy specification")
+            if for_update and len(owners_list) == 1:
```

Comment:
I think you should simply check the policy_spec.users array here instead of the owners_list. This validation is still flawed, what if I only set one user on the users array of the policy spec? The array would have users but this log would still be added.